### PR TITLE
681 add sd rtn to the core concepts page

### LINKED ISSUE #681: Add SD-RTN to the Core concepts page
Business case - users need to understand this core component in the Agora ecosystem
Subject Matter Expert - You should already have a description somewhere in the docs
Reviewer - @billy-the-fish 
Scope: Core concepts doc

### PR DESCRIPTION
**Pull request by saudsami**
closes #681

### CODE REVIEW

File: shared/voice-sdk/get-started/_get-started-sdk.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import ProjectImplement from '@docs/shared/voice-sdk/get-started/get-started-sdk
...
 
-Real-time audio engagement immerses people in the sounds of human connections, keeping them engaged in your <Vpl k="CLIENT"/> longer. <Vpd k="PRODUCT" /> enables you to add one-to-one or small-group voice communication with smooth, jitter-free streaming to your <Vpl k="CLIENT"/>. <Vg k="COMPANY"/>'s <Vpd k="SDK" /> makes it easy to embed real-time audio chat into web, mobile and native <Vpl k="CLIENT"/>s. Thanks to <Vg k="COMPANY"/>'s intelligent and global Software Defined Real-time Network (SD-RTN™), you can rely on the highest available audio quality.
+Real-time audio engagement immerses people in the sounds of human connections, keeping them engaged in your <Vpl k="CLIENT"/> longer. <Vpd k="PRODUCT" /> enables you to add one-to-one or small-group voice communication with smooth, jitter-free streaming to your <Vpl k="CLIENT"/>. <Vg k="COMPANY"/>'s <Vpd k="SDK" /> makes it easy to embed real-time audio chat into web, mobile and native <Vpl k="CLIENT"/>s. Thanks to <Vg k="COMPANY"/>'s intelligent and global Software Defined Real-time Network ([SD-RTN™](../overview/core-concepts#agora-sd-rtn)), you can rely on the highest available audio quality.
```
Comment (billy-the-fish) [inline shared/voice-sdk/get-started/_get-started-sdk.mdx:8]: Variable please.

File: shared/common/_core-concepts.mdx [hunk 2]
```diff
@@ -2,22 +2,25 @@
 
 Before you start writing your code, familiarize yourself with the core concepts that <Vg k="COMPANY" /> SDKs are built on.
 
+## Agora SD-RTN™
+
+Agora's core engagement services are powered by its Software-Defined Real-time Network (SD-RTN™) that is accessible and available anytime, anywhere around the world. The software-defined network isn’t confined by device, phone numbers, or a telecommunication provider’s coverage area like traditional networks. Agora’s global <Vg k="AGORA_BACKEND" /> is made up of over 200 data centers dedicated to delivering sub-second latency and high-availability of real-time video and audio anywhere on the globe. With <Vg k="AGORA_BACKEND" />, Agora can deliver live user engagement experiences in the form of real-time communication (RTC) with the following advantages:
```
Comment (billy-the-fish) [inline shared/common/_core-concepts.mdx:7]: Variable please.

File: shared/video-sdk/_get-started-uikit.mdx [hunk 3]
```diff
@@ -31,7 +31,7 @@ To use <Vg k="UIK" />, you implement the following steps in your app:
 
 2.  **Join a channel**
 
-    When you execute the `join` command, your <Vpl k="CLIENT" /> joins the channel, publishes the local audio and video streams to <Vg k="AGORA_BACKEND" />, and handles audio and video for anyone else who joins the call in a default UI.
+    When you execute the `join` command, your <Vpl k="CLIENT" /> joins the channel, publishes the local audio and video streams to [<Vg k="AGORA_BACKEND" />](../overview/core-concepts#agora-sd-rtn), and handles audio and video for anyone else who joins the call in a default UI.
```
Comment (billy-the-fish) [inline shared/video-sdk/_get-started-uikit.mdx:34]: Please double check this link in a running server to ensure it is good.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Nice doc, just a couple of small changes please.

File: shared/video-sdk/_get-started-uikit.mdx (hunk 3 above)
Comment (saudsami) [reply shared/video-sdk/_get-started-uikit.mdx:34]: Checked. The link is working

File: shared/common/_core-concepts.mdx (hunk 2 above)
Comment (saudsami) [reply shared/common/_core-concepts.mdx]: done

File: shared/voice-sdk/get-started/_get-started-sdk.mdx (hunk 1 above)
Comment (saudsami) [reply shared/voice-sdk/get-started/_get-started-sdk.mdx]: done

**Review by billy-the-fish — APPROVED** [review]
LGTM.

**Pull request merged**